modules/Bio/EnsEMBL/Variation/Utils/BaseVepPlugin.pm

```diff
@@ -326,6 +326,160 @@ sub check_variant_feature_type {
     return $self->_check_types('variant_feature', $type);
 }
 
+=head2 _create_vf
+
+  Arg[1]     : hash
+  Description: Create variant feature from a variant hash
+  Returntype : Bio::EnsEMBL::Variation::VariationFeature
+  Status     : Experimental
+
+=cut
+
+sub _create_vf {
+  my ($self, $var) = @_;
+
+  my $slice = Bio::EnsEMBL::Slice->new(
+    -seq_region_name  => $var->{chr},
+    -start            => $var->{start},
+    -end              => $var->{end},
+    -strand           => $var->{strand},
+  );
+
+  my $vf = Bio::EnsEMBL::Variation::VariationFeature->new(
+    -variation_name        => $var->{variation_name},
+    -seq_region_name       => $var->{chr},
+    -start                 => $var->{start},
+    -end                   => $var->{end},
+    -slice                 => $slice,
+    -strand                => $var->{strand},
+    -allele_string         => $var->{allele_string},
+    -is_somatic            => $var->{somatic},
+    -clinical_significance => $var->{clin_sig} ? [split /,/, $var->{clin_sig}] : []
+  );
+  return $vf;
+}
+
+=head2 _fetch_cache_vars
+
+  Arg[1]     : $seq_region_name
+  Arg[2]     : $start
+  Arg[3]     : $end
+  Description: Fetch variant features within a given genomic region from VEP
+               cache; supports indexed (faster) and non-indexed (slower) cache
+  Returntype : arrayref of Bio::EnsEMBL::Variation::VariationFeature
+  Status     : Experimental
+
+=cut
+
+sub _fetch_cache_vars {
+  my ($self, $chr, $start, $end) = @_;
+
+  my ($as, $variants);
+  if (defined($as = $self->_get_AnnotationSource('VariationTabix'))) {
```

Comment:
Hi @nuno-agostinho , should this `_get_AnnotationSource` should also be part of `BaseVepPlugin.pm`. It seems to generic to be in Paralogues plugin only